internal/handlers/sqlite/msg_explain.go

```diff
@@ -83,6 +83,34 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		qp.Filter = params.Filter
 	}
 
+	// Skip sorting if there are more than one sort parameters
+	if h.EnableSortPushdown && params.Sort.Len() == 1 {
+		iter := params.Sort.Iterator()
+		defer iter.Close()
+
+		var order types.SortType
+
+		var k string
+		var v any
+
+		k, v, err = iter.Next()
+		if err != nil {
+			return nil, lazyerrors.Error(err)
+		}
+
+		order, err = common.GetSortType(k, v)
+		if err != nil {
+			return nil, err
+		}
+
+		qp.Sort = &backends.SortField{
+			Key:        k,
+			Descending: order == types.Descending,
+		}
+	}
+
+	qp.Limit = params.Limit
```

Comment:
Why is logic related to limit pushdown omitted?
https://github.com/FerretDB/FerretDB/blob/093be9878f3c6b358029648c8c57d56102460ed2/internal/handlers/pg/msg_explain.go#L70-L76